src/net.h

```diff
@@ -116,13 +116,7 @@ struct CSerializedNetMsg
     std::string m_type;
 };
 
-const std::vector<std::string> CONNECTION_TYPE_DOC{
-    "outbound-full-relay (default automatic connections)",
-    "block-relay-only (does not relay transactions or addresses)",
-    "inbound (initiated by the peer)",
-    "manual (added via addnode RPC or -addnode/-connect configuration options)",
-    "addr-fetch (short-lived automatic connection for soliciting addresses)",
-    "feeler (short-lived automatic connection for testing addresses)"};
+extern const std::vector<std::string> CONNECTION_TYPE_DOC;
```

Comment:
f476fbdf84c88b7ab90aa3e9240b39806525a8b6

not sure if this was asked before, `CONNECTION_TYPE_DOC` is used only in src/rpc/net.cpp, why not just have it there?

Reply:
hmm, good point! 

history: Marco suggested extracting in https://github.com/bitcoin/bitcoin/pull/19725#discussion_r470944955. I thought a benefit was to keep `CONNECTION_TYPE_DOC` next to `ConnectionType` definition for future updates, but now we don't have that regardless. I've moved into `src/rpc/net.cpp` and added a comment on the `ConnectionType` enum for future code changes.